ext/opentelemetry-ext-pymongo/src/opentelemetry/ext/pymongo/__init__.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, tracer):
         self._span_dict = {}
 
     def started(self, event: monitoring.CommandStartedEvent):
+        """ TODO """
```

Comment:
These are explained here: https://api.mongodb.com/python/current/api/pymongo/monitoring.html

You can add a simple comment like "Method to handle a pymongo CommandStartedEvent"

Reply:
Cool thanks. Adding them as suggestions here:

```suggestion
        """ Method to handle a pymongo CommandStartedEvent """
```